packages/api-explorer/src/JsonForm/resolveRootRef.js

```diff
@@ -0,0 +1,26 @@
+import get from 'lodash.get'
+import {omit} from 'ramda'
+
+export default function resolveRootRef(schema, count = 0) {
+  if (count === 8) {
```

Comment:
Should we add a constant with a clear name

Reply:
done